handlers/admin/data-contributions.js

```diff
@@ -16,23 +19,25 @@ function editDataContribution(request, reply) {
       request.yar.flash('success', 'Data Contribution updated successfully.');
       return reply.redirect('/admin/data-contributions');
     })
-    .catch(() => {
-      // FIXME: Log error
+    .catch((error) => {
+      console.error(error.detail); // eslint-disable-line no-console
       request.yar.flash('error', 'An internal error occurred, please try again later.');
       return reply.redirect('/admin/data-contributions');
     });
 }
 
 function listDataContributions(request, reply) {
-  return new DataContribution()
-    .query('orderBy', 'created_at', 'desc')
-    .fetchAll({ withRelated: DataContribution.relatedModels })
-    .then((dataContributions) => {
-      reply.view('admin/data-contributions', {
-        title: 'Data contributions',
-        dataContributions: dataContributions.toJSON(),
-      });
-    });
+  return new DataCategory().orderBy('name').fetchAll()
+    .then((categories) => new DataContribution()
+          .query('orderBy', 'created_at', 'desc')
+          .fetchAll({ withRelated: DataContribution.relatedModels })
+          .then((dataContributions) => {
+            reply.view('admin/data-contributions', {
+              title: 'Data contributions',
+              dataContributions: dataContributions.toJSON(),
+              categories: categories.toJSON(),
+            });
+          }));
```

Comment:
I think using `Promise.join()` can improve the readability here, like:

```javascript
return Promise.join(
  new DataCategory().orderBy('name').fetchAll(),
  new DataContribution()
    .query('orderBy', 'created_at', 'desc')
    .fetchAll({ withRelated: DataContributions.relatedModels }),
  (categories, dataContributions) => {
    reply.view(/* ... */)
  })
```

You could even go a step further and extract the `.fetchAll()` calls as constants like:

```javascript
const fetchDataCategories = new DataCategory()
  .orderBy('name')
  .fetchAll();
const fetchDataContributions = new DataContribution()
  .query('orderBy', 'created_at', 'desc')
  .fetchAll({ withRelated: DataContributions.relatedModels });

return Promise.join(
  fetchDataCategories,
  fetchDataContributions,
  (categories, dataContributions) => {
    reply.view(/* ... */)
  })
```